client/src/pages/FavoriteDestinantions.js

```diff
@@ -1,11 +1,34 @@
-import React from "react";
+import React, {useState, useEffect} from "react";
 import { Typography, Grid, Button } from "@material-ui/core";
-import places from "../database/places";
 import useStyles from "../styles/FavoriteDestinations";
 import FavoriteCheckBox from "../components/FavoriteCheckBox";
 
 function FavoriteDestinantions() {
   const classes = useStyles();
+  const [places, setPlaces] = useState([])
+
+
+
+  useEffect(() => {
+    async function getData() {
+      try {
+        const cityListResponse = await fetch("/api/cities", {
+          method: "GET",
+          headers: {
+            "Content-Type": "application/json",
+          },
+        });
+        const cityList = await cityListResponse.json();
+        setPlaces(cityList);
+      } catch (error) {
+        console.error();
+      }
+    }
+    getData();
+  }, []);
```

Comment:
this component is not part of this feature so we shouldn't be editing it here

Reply:
Held of on adding anything cause I wanted to make sure my code matched with Shays before doing anything.